README.md

```diff
@@ -124,8 +124,8 @@ object_client.notify_goobi
 object_client.version.inventory
 object_client.version.current
 object_client.version.openable?(**params)
-object_client.version.open(**params)
-object_client.version.close(description: 'Changed title', significance: 'minor')
+object_client.version.open(description: 'Changed title', significance: 'minor')
+object_client.version.close(**versioning_params)
```

Comment:
I don't understand why you're expanding open (that looks good) and then obfuscating the params sent to close.  Why not expand both?